test/e2e/network/dns.go

```diff
@@ -321,6 +321,55 @@ var _ = SIGDescribe("DNS", func() {
 		validateTargetedProbeOutput(f, pod3, []string{wheezyFileName, jessieFileName}, svc.Spec.ClusterIP)
 	})
 
+	It("should support configurable pod DNS nameservers", func() {
+		By("Creating a pod with dnsPolicy=None and customized dnsConfig...")
+		testServerIP := "1.1.1.1"
+		testSearchPath := "resolv.conf.local"
+		testAgnhostPod := f.NewAgnhostPod(f.Namespace.Name, "pause")
+		testAgnhostPod.Spec.DNSPolicy = v1.DNSNone
+		testAgnhostPod.Spec.DNSConfig = &v1.PodDNSConfig{
+			Nameservers: []string{testServerIP},
+			Searches:    []string{testSearchPath},
+		}
+		testAgnhostPod, err := f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(testAgnhostPod)
+		Expect(err).NotTo(HaveOccurred(), "failed to create pod: %s", testAgnhostPod.Name)
+		framework.Logf("Created pod %v", testAgnhostPod)
+		defer func() {
+			framework.Logf("Deleting pod %s...", testAgnhostPod.Name)
+			if err := f.ClientSet.CoreV1().Pods(f.Namespace.Name).Delete(testAgnhostPod.Name, metav1.NewDeleteOptions(0)); err != nil {
+				framework.Failf("Failed to delete pod %s: %v", testAgnhostPod.Name, err)
+			}
+		}()
+		Expect(f.WaitForPodRunning(testAgnhostPod.Name)).NotTo(HaveOccurred(), "failed to wait for pod %s to be running", testAgnhostPod.Name)
+
+		// TODO: Figure out a better way other than checking the actual resolv,conf file.
```

Comment:
`resolv,conf` -> `resolv.conf`
if you wish to assign the TODO to yourself the syntax is `TODO(bclau): ...`

Reply:
I actually forgot to delete this line.